include/intx/intx.hpp

```diff
@@ -311,6 +311,17 @@ struct uint<128>
     friend constexpr bool operator>(uint x, uint y) noexcept { return y < x; }
     friend constexpr bool operator>=(uint x, uint y) noexcept { return !(x < y); }
 
+    friend constexpr std::strong_ordering operator<=>(uint x, uint y) noexcept
+    {
+        if (x < y)
```

Comment:
I think I prefer the variant where `==` is checked first and the `return (x < y) ? less : greater`.